### DISCUSSION TITLE
Need more documents and samples, thank you!

### DISCUSSION THREAD

**Discussion by markqiu** [idom-team/idom#952] [General]
Greate project, but need more documents and samples, thank you!

**Comment by Archmonger** [comment]
I've been slowly working on [rewriting the docs](https://github.com/Archmonger/idom-docs). My free time for the last 2 months has been stretched thin, but I will be picking up work on this within the next two weeks.

The concept of this rewrite is a 1:1 copy of the [beta ReactJS](https://beta.reactjs.org/learn) docs using ReactPy's syntax.

Let me know if you have any suggestions on examples that don't exist in the beta ReactJS docs, but would have liked to see when testing IDOM!

**Reply by markqiu** [reply]
Thank you very much.
What I care about: 
1. Examples of how to integrate other JS components. For example, whether you can integrate perspectives, highcharts, plotly, etc. 
3. How do I integrate custom components?

If there is enough basic documentation, I would also like to help implement some examples. Hope I can help in the future!

**Reply by Archmonger** [reply]
Yeah - The documentation about ReactPy's JavaScript integration is [severely lacking](https://github.com/reactive-python/reactpy/issues/647).

Feel free to post more comments on this thread if you think of additional docs ideas!

**Reply by numpde** [reply]
My question #978 is related.

Also, I previously built [an app](https://i.ibb.co/7kmKRDw/Screenshot-from-2023-03-10-23-38-55.png) somewhat similar to LangChain with crosstalk between reactpy/idom and vis.js, and that was unsustainably cumbersome. Would love to see more of those docs.
